pkg/agent/cniserver/server.go

```diff
@@ -536,6 +613,18 @@ func (s *CNIServer) reconcile() error {
 	return s.podConfigurator.reconcile(pods.Items)
 }
 
+func (s *CNIServer) checkAndConfigurePolicyOnlyMode() error {
```

Comment:
this going back to l2/l3 mode auto-detection where l2 or l3 CNI topo cannot be detected on init time. 

I have removed all of those auto-detect logic, but leave this here because CNI server knows about Pod configuration, therefore should be the authoritative source to determine what iptable, route table, and ovs flows should behave.

Reply:
I do not agree - otherwise other initial route, iptables, and flows should be all moved to CNI server. How policy-only is different from other modes in this front? The current CNI server just configures interfaces and does not know anything about encapMode?

Reply:
sounds good, will change